### ISSUE TITLE
Add 1.13.0, 1.13.1, 1.13.2, 1.13.3, 1.14.0, 1.14.1 & 1.14.2

### ISSUE THREAD

**Issue by mgol** [components/jqueryui#70]
Hey, we've just released jQuery UI 1.13.0: https://blog.jqueryui.com/2021/10/jquery-ui-1-13-0-released/

**Comment by liuxinghai-git** [comment]
when will it be updated

**Comment by sscotti** [comment]
Correct.  I wanted to update to 1.13 because it fixes a few bugs that were in the previous version.  Is there a repo updated for that with components or npm?

**Comment by liuxinghai-git** [comment]
when? update

**Comment by mgol** [comment]
We've released 1.13.1 now: https://blog.jqueryui.com/2022/01/jquery-ui-1-13-1-released/.

**Comment by dmmkr** [comment]
@vendethiel @RobLoach @mrohnstock , Can the jquery-ui shims repo support 1.13.0, 1.13.1, as Apache Hadoop and other related components are willing to upgrade due the following CVE's CVE-2021-41182,  CVE-2021-41183,  CVE-2021-41184 reported in 1.12.1

**Comment by vendethiel** [comment]
Sorry, not maintaining this repo anymore

**Comment by mgol** [comment]
We've released 1.13.2 now: https://blog.jqueryui.com/2022/07/jquery-ui-1-13-2-released/

**Comment by LucasAnselmoSilva12345** [comment]
Hi Devs,

When will this repository be upgrading to version 1.13.2?

**Comment by RobLoach** [comment]
Volunteering Lucas? :wink:

**Comment by marclaporte** [comment]
@AngeloJMK is looking into this now.

@RobLoach : Is there any docs or we should start some?

**Comment by RobLoach** [comment]
Don't think there's docs. Could write them if you want. Each shim repo should have the package management metadata, along with the built assets. Some repos have a `Makefile` which makes it easier to update by running `make`, but not all do.

**Comment by marclaporte** [comment]
Ok, thanks.

We will create and link from here: https://github.com/components/components/blob/master/README.md

**Comment by mgol** [comment]
We've released 1.13.3 now: https://blog.jqueryui.com/2024/04/jquery-ui-1-13-3-released/

**Comment by mgol** [comment]
We've released 1.14.0 now: https://blog.jqueryui.com/2024/08/jquery-ui-1-14-0-released/

**Comment by mgol** [comment]
We've released 1.14.1 now: https://blog.jqueryui.com/2024/10/jquery-ui-1-14-1-released/

**Comment by mgol** [comment]
We've released 1.14.2 now: https://blog.jqueryui.com/2026/01/jquery-ui-1-14-2-released/
